index.js

```diff
@@ -97,26 +147,30 @@ function myMap(array, callback) {
   return result
 }
 
-// ***** Callbacks - Question 1 *****
+function triple(num) {
+  return (num * 3)
+}
 
-// *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
 
-// console.log(myMap([1,2,3,4,5], triple)) 
+console.log(myMap([1,2,3,4,5], triple)) 
 // // => [3,6,9,12,15]
 
-// console.log(myMap([2,4,6,8], triple))   
+console.log(myMap([2,4,6,8], triple))   
 // // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 // ***** Callbacks - Question 2 *****
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
+console.log("%cCallbacks - Question 2", "color: red")
+
+function greet(name) {
+  return `Hi, ${name}!`
+}
 
-// console.log(myMap(["Raffy", "Chase"], greet)) 
+console.log(myMap(["Raffy", "Chase"], greet)) 
 // // => ["Hi, Raffy!", "Hi, Chase!"]
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
```

Comment:
Your on the right track. I noticed you didn't get to the last two, don't worry. There will be plenty more time to practice these concepts.

Look over the solutions and see if you can understand what the code is doing